transport/http/config.go

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
this package is not in /x/ but the config is, we should probably have both in an x folder (or both out of an x folder)